pyomo/contrib/incidence_analysis/tests/test_interface.py

```diff
@@ -1791,6 +1791,64 @@ def test_linear_only(self):
         self.assertIs(matching[m.eq2], m.x[2])
         self.assertIs(matching[m.eq3], m.x[3])
 
+    def test_add_edge(self):
+        m = pyo.ConcreteModel()
+        m.x = pyo.Var([1, 2, 3, 4])
+        m.eq1 = pyo.Constraint(expr=m.x[1] ** 2 + m.x[2] ** 2 + m.x[3] ** 2 == 1)
+        m.eq2 = pyo.Constraint(expr=m.x[2] + pyo.sqrt(m.x[1]) + pyo.exp(m.x[3]) == 1)
+        m.eq3 = pyo.Constraint(expr=m.x[3] + m.x[2] + m.x[4] == 1)
+        m.eq4 = pyo.Constraint(expr=m.x[1] + m.x[2] ** 2 == 5)
+
+        igraph = IncidenceGraphInterface(m, linear_only=False)
+        n_edges_original = igraph.n_edges
+
+        # Test edge is added between previously unconnectes nodes
```

Comment:
```suggestion
        # Test edge is added between previously unconnected nodes
```

Reply:
Reported in the February crate-ci/typos update PR.